tests/test_integration.py

```diff
@@ -378,3 +378,362 @@ def test(self, soco):
             'No difference between '
             'queues before and after removing the last item')
         assert len(new_queue) == len(old_queue) - 1
+
+
+class TestReorderSonosPlaylist(object):
+    """ Integration tests for Sonos Playlist Management. """
+    existing_playlists = None
+    pl_name = 'zSocoTestPlayList42'
```

Comment:
also for test code about writing variable names out, I will not comment any further on this